modules/menu/main/index.coffee

```diff
@@ -223,11 +223,16 @@ class Menu extends hx.EventEmitter
     @options.dropdownOptions.ddClass = 'hx-menu ' + if colorClass? then 'hx-' + colorClass else @options.dropdownOptions.ddClass
 
     dropdownContent = (node) ->
+      elem = hx.select(node)
+      menuItems = elem.select('.hx-menu-items')
+      if menuItems.empty()
+        menuItems = elem.append('div').class('hx-menu-items')
```

Comment:
Populate menu items inside the `hx-menu-items` div inside the dropdown. This is so we can add content outside the menu items and still re-render the menu items.